web-client/src/components/PopUpForgotPassword.jsx

```diff
@@ -0,0 +1,177 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import * as yup from "yup";
+import { useForm } from "react-hook-form";
+import { yupResolver } from "@hookform/resolvers/yup";
+import Dialog from "@material-ui/core/Dialog";
+import Snackbar from "@material-ui/core/Snackbar";
+import MuiAlert from "@material-ui/lab/Alert";
+
+const styles = makeStyles({
+  main: {
+    display: "inline-block",
+  },
+  content: {
+    padding: "30px 100px",
+  },
+  form: {
+    display: "flex",
+    flexDirection: "column",
+    float: "left",
+    textAlign: "left",
+    maxWidth: "400px",
+    margin: "16px",
+  },
+  textField: {
+    marginBottom: "32px",
+    width: "300px",
+  },
+  send: {
+    width: "100px",
+    textTransform: "none",
+    float: "right",
+    marginLeft: "auto",
+    display: "block",
+  },
+  textButton: {
+    textTransform: "none",
+  },
+  message: {
+    textAlign: "center",
+    marginTop: "30px",
+    marginBottom: "30px",
+  },
+});
+
+function Alert(props) {
+  return <MuiAlert elevation={6} {...props} />;
+}
+
+export default function PopUpForgotPassword(props) {
+  const [email, setEmail] = React.useState("");
```

Comment:
I suggest to write `useState` instead `React.useState`, because we are using this style in other files